apps/settings/lib/Search/SectionSearch.php

```diff
@@ -74,17 +74,33 @@ public function getOrder(string $route, array $routeParameters): int {
 	public function search(IUser $user, ISearchQuery $query): SearchResult {
 		$isAdmin = $this->groupManager->isAdmin($user->getUID());
 
+		$personalSections = $this->settingsManager->getPersonalSections();
+		foreach ($personalSections as $priority => &$sections) {
+			$sections = array_values(array_filter(
```

Comment:
```suggestion
		foreach ($personalSections as $priority => $sections) {
			$personalSections[$priority] = array_values(array_filter(
```
Please avoid using refs in loops, nasty stuff can happen if you are not careful.